README.md

```diff
@@ -1,3 +1,295 @@
+  zones:                                            # Remove zones block for providers other than GCP or AWS
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-b
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-c     
+    - <major region>-<minor region>-<zone>          # Example: europe-west1-d
+  credentials:                                      # credentials to get access to the seed cluster through service account
+```
+Contains the information where Gardener will create seed clusters. By default, the *initial* seed component will create a seed resource using your base cluster as seed cluster.
+
+| Field | Type | Description | Examples |Iaas Provider Documentation |
+|:------|:--------|:--------|:--------|:--------|
+|`region`|IaaS provider specific|Region where Gardener will create seed clusters and shoot clusters. The convention to use &lt;major region&gt;-&lt;minor region&gt; does not apply to all providers.<br/><br/>In Azure, use [az account list-locations](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-list-locations) to find out the location name (`name` attribute = lower case name without spaces). | `europe-west1` (GCP)<br/><br/>`eu-west-1` (AWS) <br/><br/> `westeurope` (Azure)|[GCP (HowTo)](https://cloud.google.com/kubernetes-engine/docs/how-to/managing-clusters#viewing_your_clusters), [GCP (overview)](https://cloud.google.com/docs/geography-and-regions); [AWS (HowTo)](https://docs.aws.amazon.com/cli/latest/reference/eks/describe-cluster.html), [AWS (Overview)](https://docs.aws.amazon.com/general/latest/gr/rande.html); [Azure (Overview)](https://azure.microsoft.com/en-us/global-infrastructure/geographies/), [Azure (HowTo)](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-list-locations)|
+|`zones`|IaaS provider specific|Zones where Gardener will create seed clusters and shoot clusters. This block is only required for GCP or AWS. |`europe-west1-b` (GCP)<br/></br>|[GCP (HowTo)](https://cloud.google.com/kubernetes-engine/docs/how-to/managing-clusters#viewing_your_clusters), [GCP (overview)](https://cloud.google.com/docs/geography-and-regions); [AWS (HowTo)](https://docs.aws.amazon.com/cli/latest/reference/eks/describe-cluster.html), [AWS (Overview)](https://docs.aws.amazon.com/general/latest/gr/rande.html)|
+|`credentials`|IaaS provider specific|Service account credentials in a provider-specific format. | See table with yaml keys below. | [GCP](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys), [AWS](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users.html#id_users_service_accounts), [Azure](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)|
+
+The service account credentials will be used to give Gardener access to your base cluster:
```

Comment:
The *credentials* (service account is only for GKE) provide access to the iaas layer used for creating required resources for *shoot* clusters. They are *not* used to access the base cluster.

Reply:
@mandelsoft : 
I rewrote it, but I have a feeling that further clarification is needed.

What exactly is the "IaaS layer"? It seems to be something different for each IaaS provider so it might be helpful to name for each provider what is meant.

- GKE: service account
- AWS: ???
- Azure: ???

I am not even sure any more I linked to the right documentation here. If I new what 'IaaS layer' means in each case I could easier find a useful link in the IaaS provider's documentation.